Mage.Sets/src/mage/sets/Commander2021Edition.java

```diff
@@ -212,6 +213,7 @@ private Commander2021Edition() {
         cards.add(new SetCardInfo("Orzhov Advokist", 99, Rarity.UNCOMMON, mage.cards.o.OrzhovAdvokist.class));
         cards.add(new SetCardInfo("Orzhov Basilica", 308, Rarity.UNCOMMON, mage.cards.o.OrzhovBasilica.class));
         cards.add(new SetCardInfo("Orzhov Signet", 254, Rarity.UNCOMMON, mage.cards.o.OrzhovSignet.class));
+        cards.add(new SetCardInfo("Osgir, the Reconstructor", 8, Rarity.MYTHIC, OsgirTheReconstructor.class));
```

Comment:
replace `OsgirTheReconstructor.class` with `mage.cards.o.OsgirTheReconstructor.class` to eliminate the import

Reply:
no problem, this has been updated